examples/flutter_gallery/.idea/runConfigurations/main_dart.xml

```diff
@@ -1,5 +1,5 @@
 <component name="ProjectRunConfigurationManager">
-  <configuration default="false" name="flutter_gallery" type="FlutterRunConfigurationType" factoryName="Flutter">
+  <configuration default="false" name="main.dart" type="FlutterRunConfigurationType" factoryName="Flutter">
```

Comment:
This is consistent with the template:
https://github.com/flutter/flutter/tree/master/packages/flutter_tools/templates/create/.idea/runConfigurations

Is the template incorrect?